crates/database/src/order_rewards.rs

```diff
@@ -8,7 +8,7 @@ pub async fn save(
     reward: f64,
 ) -> Result<(), sqlx::Error> {
     const QUERY: &str = r#"
-INSERT INTO order_rewards (order_uid, auction_id, reward)
+INSERT INTO order_execution (order_uid, auction_id, reward)
```

Comment:
Won't this error because of:
```
"order_rewards_pkey" PRIMARY KEY, btree (order_uid, auction_id)
```

Since we would always insert the reward before the surplus fee?

Also - if we aren't computing the reward for limit orders, then that is a bug!